x-pack/legacy/plugins/ml/public/overview/components/anomaly_detection_panel/table.tsx

```diff
@@ -70,13 +73,39 @@ export const AnomalyDetectionTable: FC<Props> = ({ items, jobsList, statsBarData
     },
     {
       field: AnomalyDetectionListColumns.maxAnomalyScore,
-      name: i18n.translate('xpack.ml.overview.anomalyDetection.tableMaxScore', {
-        defaultMessage: 'Max anomaly score',
-      }),
+      name: (
+        <EuiToolTip
+          content={i18n.translate('xpack.ml.overview.anomalyDetection.tableMaxScoreTooltip', {
+            defaultMessage: 'The maximum score for each job in its most recent 24 hour period',
```

Comment:
Does `Maximum score across all jobs in the group over its most recent 24 hour period` sound better?

Reply:
Updated in a452f18e5ef1e497d0f4b0b77b6618b1040f556b